packages/migrate/test/fixtures/migrate/src/complex.js

```diff
@@ -1,4 +1,7 @@
 import { salutations, DEFAULT_GREETING } from './salutations';
+import { v4 as uuid } from 'uuid';
+
+uuid();
```

Comment:
Need to add Line 2-4 to packages/migrate/test/fixtures/migrate/output/complex.ts.ouput.